rpc/session_utils.c

```diff
@@ -104,6 +105,8 @@ M0_INTERNAL uint64_t m0_rpc_id_generate(void)
 		id = (millisec << 10) | (m0_atomic64_get(&cnt) & 0x3FF);
 	} while (id == 0 || id == UINT64_MAX);
 
+	id = m0_hash(m0_fid_hash(uniq_fid) + id);
```

Comment:
I think we need to move this into the above "do {} while()" loop.
In rare case, 'id' count be zero or UINT64_MAX, and we need to re-generate this id.

Reply:
Done